examples/impute/plot_missing_values.py

```diff
@@ -79,6 +83,13 @@ def get_results(dataset):
     imputer = SimpleImputer(missing_values=0, strategy="mean")
     mean_impute_scores = get_scores_for_imputer(imputer, X_missing, y_missing)
 
+    # Estimate the score after kNN-imputation of the missing values
+    knn_estimator = make_pipeline(
```

Comment:
nitpick: `knn_estimator` might be mistaken for an actual `KNeighborsRegressor` ... maybe `knn_impute_estimator`?

Reply:
How about something like 'knnimpute_and_rf' ?